.azure-pipelines/ultimate-pipeline.yml

```diff
@@ -922,6 +922,64 @@ stages:
     - script: tracer\build.cmd CheckBuildLogsForErrors
       displayName: Check logs for errors
 
+- stage: integration_tests_windows_debugger
```

Comment:
I don't feel strongly about it, but it seems like this is ~90% code duplication between `integration_tests_linux_debugger` and `integration_tests_linux_debugger`, `integration_tests_windows` and `integration_tests_windows`. Could we have potensially avoided this by extracting out a template to a separate `yml` file and passing in the name of the integration tests project as a parameter?

Reply:
May be a good idea for future optimizations.